### PR TITLE
Update visit 231

### PR DESCRIPTION
**Pull request by hhummel**
Backend updates visit for notes, urgency and program/service queue

### CODE REVIEW

File: core/tests/visits.py [hunk 1]
```diff
@@ -77,6 +80,86 @@ def test_update_visit_notes(self):
         self.assertEqual(Visit.objects.count(), 1)
         self.assertEqual(Visit.objects.get(id=visit_id).notes, new_note)
 
+    def test_update_visit_urgency(self):
+        """
+        Ensure we can update notes on a visit
```
Comment (MikeyManoguerra) [inline core/tests/visits.py]: can you update this docstring?

File: core/tests/visits.py [hunk 2]
```diff
@@ -77,6 +80,86 @@ def test_update_visit_notes(self):
...
+            "service": 1,
+            "urgency": "_2",
+        }
+        create_response = self.client.post(
+            "/api/visits/", data, format="json", **headers
+        )
+        self.assertEqual(create_response.status_code, status.HTTP_201_CREATED)
+
+        headers = self.auth_headers_for_user("case_manager")
+        new_note = "I forgot to add notes the first time!"
```
Comment (MikeyManoguerra) [inline core/tests/visits.py]: doesnt look like this is being used in this test, can you remove?

File: core/tests/visits.py (hunk 1 above)
Comment (hhummel) [reply core/tests/visits.py]: Good catch

File: core/tests/visits.py (hunk 2 above)
Comment (hhummel) [reply core/tests/visits.py]: Done

**Comment by MikeyManoguerra** [comment]
I approve

**Pull request merged**